File: atv13/exerc13.go
package main
import "fmt"

func Fatorial(n int) int {
    if n == 0 {
        return 1
    }
    resultado := 1
    for i := 1; i <= n; i++ {
        resultado *= i
    }
    return resultado
}

func main() {
    fmt.Println("Fatorial de 5:", Fatorial(5))
}

File: atv13/exerc13.py
class Matematica:
    @staticmethod
    def fatorial(n):
        if n == 0:
            return 1
        resultado = 1
        for i in range(1, n + 1):
            resultado *= i
        return resultado

print(f"Fatorial de 5: {Matematica.fatorial(5)}")
